dotnet-datetime/CheckNumberOfDaysBetweenTwoDates/CheckNumberOfDaysBetweenTwoDates/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using CheckNumberOfDaysBetweenTwoDates;
+class Program
+{
+    static void Main()

```

Comment:
You can do this file with a top-level statement, you don't need to define a class here and the `Main()` method.